### PR TITLE
Improve test stability

### PR DESCRIPTION
**Pull request by avsej**

### CODE REVIEW

**Comment by avsej** [comment]
jenkins, test this please

**Comment by avsej** [comment]
jenkins, test this please

**Review by thejcfactor — DISMISSED** [review]
As these are tests, I don't think it blocks the review, but I did have a comment on some txns tests missing a potential skip if the server doesn't support txn queries.  Otherwise LGTM 👍

File: test/test_transaction_simple.cxx
[the remark this answers is not in the document]
Comment (avsej) [reply test/test_transaction_simple.cxx:1]: good catch, those ag gren because we don't detect the reason of the rollback accurately

**Review by thejcfactor — APPROVED** [review]
LGTM 👍

**Pull request merged**